README.md

```diff
@@ -17,6 +17,21 @@ An Amazon Aurora DB cluster uses failover to automatically repairs the DB cluste
 
 Although Aurora is able to provide maximum availability through the use of failover, existing client drivers do not fully support this functionality. This is partially due to the time required for the DNS of the new primary DB instance to be fully resolved in order to properly direct the connection. The AWS JDBC Driver for MySQL fully utilizes failover behavior by maintaining a cache of the Aurora cluster topology and each DB instance's role (Aurora Replica or primary DB instance). This topology is provided via a direct query to the Aurora database, essentially providing a shortcut to bypass the delays caused by DNS resolution. With this knowledge, the AWS JDBC Driver can more closely monitor the Aurora DB cluster status so that a connection to the new primary DB instance can be established as fast as possible. Additionally, as noted above, the AWS JDBC Driver is designed to be a drop-in compatible for other MySQL JDBC drivers and can be used to interact with RDS and MySQL databases as well as Aurora MySQL.
 
+## Differences between the AWS MySQL JDBC driver and the AWS JDBC Driver
+The AWS MySQL JDBC driver, as its name explicitly says, is a MySQL JDBC driver built on top of the [MySQL-Connector-J JDBC driver](https://github.com/mysql/mysql-connector-j) to provide features and enhancements of Amazon Aurora databases to that driver. That means that a user can only use the AWS MySQL JDBC driver with MySQL databases and/or MySQL-compatible Amazon RDS and Amazon Aurora clusters.
+
+The AWS JDBC Driver (previously named AWS Advanced JDBC Wrapper) provides similar functionalities. However, it is not specific to a certain vendor database. The idea behind this driver is to build a feature/functionality layer for JDBC on top of an existing database driver. It was designed to work with multiple relational databases and also different drivers, whether they are community-based or proprietary drivers. For the databases, the AWS JDBC Driver currently supports MySQL, but also PostgreSQL and MariaDB.
```

Comment:
Lets pick either feature or functionality instead of using `feature/functionality`.
We support mysql, pg and mariadb drivers but we don't offcially support mariadb databases.
```suggestion
The [AWS JDBC Driver](https://github.com/awslabs/aws-advanced-jdbc-wrapper) provides similar functionalities. However, it is not specific to a certain vendor database. The idea behind this driver is to build a feature/functionality layer for JDBC on top of an existing database driver. It was designed to work with multiple relational databases and also different drivers, whether they are community-based or proprietary drivers. For the databases, the AWS JDBC Driver currently supports MySQL and PostgreSQL.
```

Reply:
Done